api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -877,20 +882,37 @@ def pick_up_tip(  # noqa: C901
             if self._api_version >= _PARTIAL_NOZZLE_CONFIGURATION_ADDED_IN
             else self.channels
         )
+        nozzle_map = (
+            self.nozzle_map
+            if self._api_version
+            >= _PARTIAL_NOZZLE_CONFIGURATION_AUTOMATIC_TIP_TRACKING_IN
+            else None
+        )
 
         if location is None:
+            if (
+                nozzle_map is not None
+                and nozzle_map.configuration != NozzleConfigurationType.FULL
+                and self.starting_tip is not None
+            ):
```

Comment:
I'm definitely going to forget why this was problematic to support right away. Is this a good place to leave a comment?

```suggestion
            if (
                nozzle_map is not None
                and nozzle_map.configuration != NozzleConfigurationType.FULL
                and self.starting_tip is not None
            ):
            # Disallow this to avoid having to think about the direction
            # in which self.starting_tip consumes tips. It might
            # have to vary now to match the varying orders in which
            # different pipette configurations pick up tips.
            # We might want to undeprecate the Labware.has_tip setter
            # and use that to replace starting_tip.
```